bin/mediatis-typo3-coding-standards-setup

```diff
@@ -120,24 +198,119 @@ class MediatisCodingStandardsSetup
 
     protected function setupRectorConfig(): void
     {
-        $sourceFilePath = match($this->typo3Version) {
-            11 => 'rector-typo3-11.php',
-            12 => 'rector-typo3-12.php',
-        };
-        $this->updateFile(
-            $sourceFilePath,
-            'rector.php'
-        );
+        foreach ($this->supportedTypo3PackageKeys as $key) {
+            $versionConstraint = $this->getDependencyVersionConstraintsFromComposerData($key, $this->supportedTypo3Versions, 'major');
+            if ($versionConstraint) {
+                $versionConstraint = reset($versionConstraint); // Only need lowest supported version
+                break;
+            }
+        }
+        if ($versionConstraint) {
+            $sourceFilePath = match(intval($versionConstraint)) {
+                11 => 'rector-typo3-11.php',
+                12 => 'rector-typo3-12.php',
```

Comment:
@mkarulin Would it make sense to maintain the rector file list at the top, next to `$supportedTypo3Versions` and the other variables?